click/exceptions.py

```diff
@@ -51,7 +51,9 @@ def show(self, file=None):
         color = None
         if self.ctx is not None:
             color = self.ctx.color
-            echo(self.ctx.get_usage() + '\n', file=file, color=color)
+            echo(self.ctx.get_usage() + '\nTry "%s %s" for help.\n'
+                 % (self.ctx.command_path, self.ctx.help_option_names[0]),
```

Comment:
What if there is no help option (you can disable it with `add_help_option=False`)?